client/mussel/musselrc

```diff
@@ -0,0 +1,55 @@
+# /etc/wakame-vdc/musselrc
+
+# System wide functions and aliases
+# Environment stuff goes in /etc/wakame-vdc/musselrc
+
+# It's NOT a good idea to change this file unless you know what you
+# are doing. It's much better to create a custom.sh shell script in
+# ~/.musselrc to make custom changes to your environment, as this
+# will prevent the need for merging in future updates.
+
+#MUSSEL_FRONTEND=noninteractive # [ interactive | noninteractive ]
+#MUSSEL_LOGLEVEL=info # [ info | debug ]
+#MUSSEL_DRY_RUN= #  [ y | yes | n | no ]
+
+#DCMGR_API_VERSION=12.03
+#DCMGR_HOST=127.0.0.1
+#DCMGR_PORT=9001
+#DCMGR_BASE_URI=http://${DCMGR_HOST}:${DCMGR_PORT}/api/${DCMGR_API_VERSION}
+#DCMGR_RESPONSE_FORMAT=yml
+
+account_id=a-shpoolxx
+
+# For instance
+#
+# **required**
```

Comment:
I think the \*\*required\*\* is confusing. They might be required for the WebAPI, but this is musselrc. In here, they are not required since Mussel will assign a defailt value.